Tasks/PackerBuildV1/task.json

```diff
@@ -235,8 +235,18 @@
             "label": "Skip temporary file cleanup during deprovision",
             "defaultValue": true,
             "groupName": "Advanced",
+            "visibleRule": "templateType = builtin",
             "helpMarkDown": "During deprovisioning of VM, skip clean-up of temporary files uploaded to VM. Refer [here](https://www.packer.io/docs/builders/azure.html#skip_clean)"
         },
+        {
+            "name": "packerVersion",
+            "type": "string",
+            "label": "Packer Version",
+            "required": false,
+            "defaultValue": "",
+            "groupName": "Advanced",
+            "helpMarkDown": "Specify the version of Packer to install."
```

Comment:
Lets improve the help by adding that: This will work only with custom templates.

Reply:
Changed the visibility of packerbuild version to only custom template.

Reply:
It will still be helpful to add that comment for probably yaml scenarios etc

Reply:
done